### ISSUE TITLE
How to set the maximum scale of the fontSize?

### ISSUE THREAD

**Issue by kennic** [OpenFlutter/flutter_screenutil#580]
This util library makes the font size too big on certain of devices, and we must have a way to limit the font size.

**Comment by sheheryarkhan501** [comment]
Yes, I have personally derived another way to make things better, which is using the radial value ".r", instead of using ".sp".
